codespan/src/filemap.rs

```diff
@@ -104,7 +104,11 @@ pub struct FileMap {
 
 impl FileMap {
     /// Construct a new filemap, creating an index of line start locations
```

Comment:
I wonder if we could update the documentation here, saying that this is hand for tests, but production code should probably go via a `CoadeMap` 🤔